RoaringBitmap/src/main/java/org/roaringbitmap/buffer/BufferFastAggregation.java

```diff
@@ -560,10 +556,7 @@ static MutableRoaringBitmap workShyAnd(long[] aggregationBuffer,
       Arrays.fill(words, -1L);
       MappeableContainer tmp = new MappeableBitmapContainer(LongBuffer.wrap(words), -1);
       for (MappeableContainer container : slice) {
-        MappeableContainer and = tmp.iand(container);
```

Comment:
This is an optimisation

Reply:
@richardstartin Same question ... can you explain (briefly) the optimization?

Reply:
Answered above.